"Could not open requirements file" since `requirements.txt` is in action_path, not caller workspace

```console
Run eic/generate-meeting-slides@v1
Run actions/cache@v3
Cache not found for input keys: Linux-venv-, Linux-venv-
Run python -m venv ./.venv
  python -m venv ./.venv
  . ./.venv/bin/activate
  pip install -r requirements.txt
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'
Error: Process completed with exit code 1.
```

--- a/action.yml
+++ b/action.yml
@@ -28,7 +28,7 @@ runs:
     - run: |
         python -m venv ./.venv
         . ./.venv/bin/activate
-        pip install -r requirements.txt
+        pip install -r ${{ github.action_path }}/requirements.txt
       if: steps.cache-venv.outputs.cache-hit != 'true'
       shell: bash
     - run: |

